fades/main.py

```diff
@@ -60,7 +60,7 @@ def _merge_deps(*deps):
     final = {}
     for dep in deps:
         for repo, info in dep.items():
-            final.setdefault(repo, []).extend(info)
+            final.setdefault(repo, []).extend(set(info))
```

Comment:
why not having, in final, per repo, directly a set? something like:
```
final.setdefault(repo, set()).update(info)
```

Reply:
I wasn't sure if the code is expecting to receive a list in any other place. 

I can check it later.